MonitoringSystem/ConfigTemplate.cfg

```diff
@@ -0,0 +1,15 @@
+Services
+{
+  Monitoring
+  {
+    Port = 9201
```

Comment:
Are you sure you can do that? If I remember properly, only ports 9130 to 9200 are asked to be open...